apps/web/pages/event-types/[type].tsx

```diff
@@ -1610,6 +1616,8 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                         onChange={(val) => {
                           if (val) {
                             locationFormMethods.setValue("locationType", val.value);
+                            locationFormMethods.unregister("locationLink");
```

Comment:
Un register the input so that they are not present when you are trying to change any other location